config/snekbox.cfg

```diff
@@ -104,10 +104,18 @@ mount {
     rw: false
 }
 
+mount {
+    dst: "/dev/shm"
+    fstype: "tmpfs"
+    rw: true
+    is_bind: false
+    options: "size=40m"
+}
+
 cgroup_mem_max: 52428800
 cgroup_mem_mount: "/sys/fs/cgroup/memory"
 
-cgroup_pids_max: 5
+cgroup_pids_max: 6
```

Comment:
We need to understand exactly why it has to be upped to 6. Someone showed that it spawns some extra internal threads as an implementation detail. The rest of the threads are based on `os.cpu_count` if the count is unspecified.

How consistent and reliable is that behaviour? Furthermore, should we be using `os.cpu_count` to automatically calculate what this value should be? Such feature would have to be optional, otherwise it'd force an effective minimum on the PID limit, which not all users may want.

Reply:
On second thought, using `cpu_count` for calculation sounds like tight coupling, especially considering it's also relying on an implementation detail of the number of threads the pool creates. I suppose hard-coding the PID limit in the config is fine, as long as we can ensure the `cpu_count` is consistent in the production environment.

Reply:
The test was changed to read this value from the config here c50b5473d74d6d2ef6f29eca68c65b4f61b3466d